src/v0/destinations/af/transform.js

```diff
@@ -196,7 +203,7 @@ function processSingleMessage(message, destination) {
   let payload;
   switch (messageType) {
     case EventType.TRACK: {
-      payload = processEventTypeTrack(message);
+      payload = processEventTypeTrack(message, destination.Config.placePropertiesAtRoot);
```

Comment:
More appropriate naming.
```suggestion
      payload = processEventTypeTrack(message, destination.Config.addPropertiesAtRoot);
```